src/importAssets.d.ts

```diff
@@ -17,11 +17,13 @@
 
 // https://stackoverflow.com/questions/43638454/webpack-typescript-image-import
 declare module "*.svg" {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
   const value: string;
   export default value;
 }
 
 declare module "*.txt" {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
   const value: string;
   export default value;
```

Comment:
Can't this just be made uppercase? It's not visible anywhere
```suggestion
  const CONTENT: string;
  export default CONTENT;
```

Reply:
okay